### PR TITLE
conditional rendering on Homenavbar page

### PR DESCRIPTION
**Pull request by Bit-Han**
Conditional rendering on the HomeNavbar page

### CODE REVIEW

File: src/sections/HomeNavbar.tsx [hunk 1]
```diff
@@ -1,18 +1,23 @@
 import React, { useState } from "react";
+import { useSelector } from "react-redux";
```
Comment (pablo-clueless) [inline src/sections/HomeNavbar.tsx]: Use the typed selector from `../hooks`

**Review by pablo-clueless — CHANGES_REQUESTED** [review]
Thanks for adding the false state to `isLoggedIn`.


There are typed imports available in the hooks folder. We can't use the normal useSelector and useDispatch.

**Review by taiwoAkindele — APPROVED** [review]

File: src/sections/HomeNavbar.tsx (hunk 1 above)
Comment (PhilemonBrain) [reply src/sections/HomeNavbar.tsx]: @Bit-Han did u compare with the suggestion offered by samsom?

Comment (Bit-Han) [reply src/sections/HomeNavbar.tsx]: Thanks samson. i have changed it to the typed selector and dispatch on hooks.

**Pull request merged**